proto/Cargo.toml

```diff
@@ -18,11 +18,11 @@ protobuf-codec = ["grpcio/protobuf-codec", "protobuf-build/grpcio-protobuf-codec
 prost-codec = ["prost-derive", "prost-types", "bytes", "lazy_static", "grpcio/prost-codec", "prost", "protobuf-build/grpcio-prost-codec"]
 
 [dependencies]
-grpcio = { path = "..", features = ["boringssl"], version = "0.9.0", default-features = false }
+grpcio = { path = "..", features = ["boringssl"], version = "0.10.0", default-features = false }
```

Comment:
It looks like all local libs can be used without specifying a version?

Reply:
Unless they need to be published to crates.io.

Reply:
got it, `tests-and-examples` dont need be published to crates.io so the version can be ignored.